pkg/transform/kubernetes/populators_test.go

```diff
@@ -588,6 +588,53 @@ func TestPopulateEnvFrom(t *testing.T) {
 	}
 }
 
+func TestPopulateVolumes(t *testing.T) {
+	volumeClaims := []spec.VolumeClaim{{Name: "foo"}, {Name: "bar"}, {Name: "barfoo"}}
+	volumes := []api_v1.Volume{{Name: "foo"}}
+
+	// a volumeMount is defined but that is not there in volumeClaims
+	// neither it is in pod level volumes, so this should fail
+	failingContainers := []api_v1.Container{
+		{VolumeMounts: []api_v1.VolumeMount{{Name: "baz"}}},
+	}
+
+	if _, err := populateVolumes(failingContainers, volumeClaims, volumes); err == nil {
+		t.Errorf("should have failed but passed for volumeMount that" +
+			" does not exist.")
+	}
+
+	passingContainers := []api_v1.Container{
```

Comment:
the whole passing failing this is very confusing for me :(
There are no test names, expected behavior, etc fields in any struct.
I don't get it why are we moving away from the way tests have been written all this while.
e.g. https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/cronjob/cronjob_controller_test.go